internal/server/proto/server.proto

```diff
@@ -2828,6 +2831,24 @@ message GetStatusReportRequest {
   // a report by deployment or release operation id
 }
 
+message ExpediteStatusReportRequest {
+  Ref.Operation ref = 1;
```

Comment:
It doesn’t look like we actually use this in the service. Maybe we can do without it?